Install instructions incomplete

I just did a git clone of geany. I read INSTALL for instructions: 

```
The simplest way to compile this package is:

  1. `cd' to the directory containing the package's source code and type
     `./configure' to configure the package for your system.  If you're
     using `csh' on an old version of System V, you might need to type
     `sh ./configure' instead to prevent `csh' from trying to execute
     `configure' itself.
```

 ./configure: No such file or directory

It seems that I need to run autogen.sh first. This should not need to be guessed or known from experience elsewhere. 

Also: 

./configure  --help 
```
  --enable-html-docs      generate HTML documentation using rst2html
                          [default=auto]
```

The auto fn does not seem to work since this causes a configure error and make files are not produced.
```
configure: error: Documentation enabled but rst2html not found.
You can explicitly disable building of the HTML manual with --disable-html-docs,
but you then may not have a local copy of the HTML manual.
```

--- a/README
+++ b/README
@@ -46,6 +46,9 @@ $ make
 (as root)
 % make install
 
+If you are building from a Git clone rather than a source tarball, you need
+to run ./autogen.sh first.
+
 For more configuration details run
 $ ./configure --help
 

